.circleci/config.yml

```diff
@@ -96,11 +96,26 @@ jobs:
       - restore_cache:
           key: v1-tree-{{ .Environment.CIRCLE_SHA1 }}
       - run:
-          name: Test simulation
+          name: Test individual module simulations
           command: |
             export PATH="$GOBIN:$PATH"
-            export GAIA_SIMULATION_SEED=1531897442166404087
-            make test_sim
+            make test_sim_modules
+
+  test_sim_gaia_fast:
+    <<: *defaults
+    parallelism: 1
+    steps:
+      - attach_workspace:
+          at: /tmp/workspace
+      - restore_cache:
+          key: v1-pkg-cache
+      - restore_cache:
+          key: v1-tree-{{ .Environment.CIRCLE_SHA1 }}
+      - run:
+          name: Test full Gaia simulation
```

Comment:
Is this a full simulation or a fast one?

Reply:
It tests them together though, rather than individually (that's `test_sim_modules`).